buildmodel/commands.go

```diff
@@ -148,6 +151,18 @@ func BindPRFlags() *PRFlags {
 // submits the resulting commit as a GitHub PR, approves with a second account, and enables the
 // GitHub auto-merge feature.
 func SubmitUpdatePR(f *PRFlags) error {
+	var auther gitcmd.URLAuther = gitcmd.NoAuther{}
+	if f.githubPAT != nil {
+		auther = &gitcmd.GitHubPATAuther{
+			PAT: *f.githubPAT,
+		}
+	}
+	var reviewAuther gitcmd.URLAuther = gitcmd.NoAuther{}
+	if f.githubPATReviewer != nil {
+		reviewAuther = &gitcmd.GitHubPATAuther{
+			PAT: *f.githubPATReviewer,
+		}
+	}
```

Comment:
Could these be centralized on `GHAuthFlags` to avoid duplication?

Reply:
updated